readthedocs/organizations/views/private.py

```diff
@@ -97,15 +99,25 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class AddOrganizationOwner(PrivateViewMixin, OrganizationOwnerView, CreateView):
-    template_name = 'organizations/admin/owners_edit.html'
-    success_message = _('Owner added')
+class AddOrganizationOwner(PrivateViewMixin, OrganizationOwnerView, FormView):
+    template_name = "organizations/admin/owners_edit.html"
+    success_message = _("Invitation sent")
+
+    def form_valid(self, form):
+        form.save()
+        return super().form_valid(form)
 
 
 class DeleteOrganizationOwner(PrivateViewMixin, OrganizationOwnerView, DeleteView):
     success_message = _('Owner removed')
     http_method_names = ['post']
 
+    def post(self, request, *args, **kwargs):
+        if self._is_last_user():
+            return HttpResponseBadRequest(_("User is the last owner, can't be removed"))
```

Comment:
Shouldn't this be a Form ValidationError instead?

Reply:
Users should never see this if they use web interface, only if they do the request manually (we don't show the option if you are the only user left).